assets/src/block-editor/QueryLoopBlockExtension/index.js

```diff
@@ -108,6 +108,20 @@ export const setupQueryLoopBlockExtension = () => {
           wp.data.select('core/block-editor').getSelectedBlock(),
         ]);
 
+        useEffect(() => {
+          const IS_NEW_IA = window.p4_vars.options.new_ia;
+          const queryPostType = IS_NEW_IA ? 'p4_action' : 'page';
+
+          if(query.postType !== queryPostType) {
```

Comment:
Need to implement this since the query passed through attributes (1) it's not the same as the one needed by the query loop (2)

For instance:
(1)
```json
{
   "post_type":"page",
   "order":"DESC",
   "orderby":"date",
   "post__not_in":[
      2067
   ],
   "tax_query":[
      
   ],
   "offset":0,
   "posts_per_page":3,
   "author__in":[]
}
```
(2)
```json
{
   "pages":0,
   "perPage":3,
   "offset":0,
   "order":"desc",
   "orderBy":"date",
   "postStatus":"publish",
   "author":"",
   "search":"",
   "exclude":[
      2067
   ],
   "sticky":"",
   "inherit":false,
   "postType":"page",
   "postIn":[
      
   ],
   "hasPassword":false,
   "parent":9,
   "post_parent":9
}
```